docs/framework/20categories/20architecture/liquidity-networks.md

```diff
@@ -1,3 +1,24 @@
 ### Liquidity Networks
+Liquidity networks enable users to transfer already minted assets from one chain to another. With such protocols, there's no ‘bridging’ of assets, i.e., no assets move across chains. Instead, assets are exchanged between one user (liquidity providers, aka LPs) and another bridge user on different networks, resulting in a cross-chain swap. For example, a user receives the desired asset on the destination network via the LP. In return, the LP receives the user’s assets on the source network and fees for providing the desired asset. 
 
-Note: include HTLCs as a possible design approach
\ No newline at end of file
+Typically, LP funds within a liquidity network are collected and locked in a smart contract (liquidity pool) to facilitate quick cross-chain swaps. When a user initiates a transaction, the desired assets are released from the smart contract on the destination network. In essence, liquidity networks work similarly to peer-to-peer networks where LPs, acting as routers, own and distribute liquidity across chains. 
```

Comment:
Is this typically, or always?   I think it is always. In this case, I would remove "Typically, ".